website/src/app/[lang]/[region]/(website)/partners/(sections)/ngocard.tsx

```diff
@@ -0,0 +1,371 @@
+							<Badge className="bg-primary hover:bg-primary text-primary space-x-2 bg-opacity-10 px-4 py-2 hover:bg-opacity-100 hover:text-white">
+								{countryBadge?.countryFlagComponent || <SL className="h-5 w-5 rounded-full" />}
+								<Typography size="md" weight="normal" className="text-inherit">
+									{translator.t(countryBadge?.countryAbbreviation || 'SL')}
+								</Typography>
+							</Badge>
+						</div>
+					</div>
+					{ngoHoverCard.orgDescriptionParagraphs.map((paragraph, index) => {
+						return (
+							<div key={index} className="mb-4">
+								{paragraph.map((fragment, index2) => {
+									return fragment.href ? (
+										<Link href={fragment.href} key={index2}>
+											<Typography as="span" size="lg" color="primary">
+												{fragment.text}
+											</Typography>
+										</Link>
+									) : (
+										<Typography as="span" size="lg" key={index}>
+											{fragment.text}
+										</Typography>
+									);
+								})}
+							</div>
+						);
+					})}
+					<Separator className="bg-primary my-6 bg-opacity-10" />
+					{ngoHoverCard.quote && ngoHoverCard.quoteAuthor ? (
+						<>
+							<div className="py-12 text-center">
+								<div className="my-4 px-6">
+									{ngoHoverCard.quote.map((title, index) => (
+										<Typography as="span" size="3xl" weight="medium" color={title.color} key={index}>
+											{title.text}{' '}
+										</Typography>
+									))}
+								</div>
+								<div className="my-4">
+									<Typography size="lg">
+										{ngoHoverCard.quoteAuthor}, {orgShortName}
+									</Typography>
+								</div>
+							</div>
+							<Separator className="bg-primary my-6 bg-opacity-10" />
+						</>
+					) : (
+						''
+					)}
+					<div className="grid grid-cols-3 gap-4">
+						<div className="col-span-1">
+							<Typography size="lg">{translator.t('ngo-generic.mission')}</Typography>
+						</div>
+						<div className="col-span-2">
+							<Typography size="lg">{orgMission}</Typography>
+						</div>
+					</div>
+					<div className="grid grid-cols-3 gap-4">
+						<div className="col-span-1">
+							<Typography size="lg">{translator.t('ngo-generic.founded')}</Typography>
+						</div>
+						<div className="col-span-2">
+							<Typography size="lg">{ngoHoverCard.orgFoundation}</Typography>
+						</div>
+					</div>
+					<div className="grid grid-cols-3 gap-4">
+						<div className="col-span-1">
+							<Typography size="lg">{translator.t('ngo-generic.headquarter')}</Typography>
+						</div>
+						<div className="col-span-2">
+							<Typography size="lg">{ngoHoverCard.orgHeadquarter}</Typography>
+						</div>
+					</div>
+					{showVisitOnline ? (
+						<div className="grid grid-cols-3 gap-4">
+							<div className="col-span-1">
+								<Typography size="lg">{translator.t('links.more')}</Typography>
+							</div>
+							<div className="col-span-2">
+								{ngoHoverCard.orgWebsite ? (
```

Comment:
Instead of 

```ts
{ngoHoverCard.orgWebsite ? (
	<Link href={ngoHoverCard.orgWebsite} className="ml-auto inline-block pr-2 text-lg underline">
	{translator.t('links.website')}
	</Link>
) : (
		''
)}
```
you can do

```ts
{ngoHoverCard.orgWebsite && (
	<Link href={ngoHoverCard.orgWebsite} className="ml-auto inline-block pr-2 text-lg underline">
	{translator.t('links.website')}
	</Link>
)
```

which is a little cleaner imo :)

Reply:
Resolved